new_client/src/App.js

```diff
@@ -1,21 +1,21 @@
-import React from 'react';
-import { Router, Route, Switch } from 'react-router-dom';
+import React from "react";
+import { Router, Route, Switch } from "react-router-dom";
 import { createBrowserHistory } from "history";
 import { Provider } from "react-redux";
 import configureStore from "./store/configureStore";
 
 // Components begin here
-import Header from "./components/Header"
-import Home from './components/Home/home';
-import Footer from './components/Footer'
-import About from './components/pages/About';
-import Glimpse from './components/glimpses/Glimpse';
+import Header from "./components/Header";
+import Home from "./components/Home/home";
+import Footer from "./components/Footer";
+import About from "./components/pages/About";
+import Glimpse from "./components/glimpses/Glimpse";
 
-import Blogs from './components/blogs/Blog';
-import AddBlog from './components/blogs/AddBlog';
+import Blogs from "./components/blogs/Blog";
```

Comment:
Keeping uniformity by writing .jsx extension with all the files imported.

Reply:
Keeping uniformity is right. But I believe lets instead remove the '.jsx' extension from the other imports.

Reply:
Yes,Okay.